sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/client.py

```diff
@@ -59,14 +59,14 @@ class AMQPClient(object):
 
     :param remote_address: The AMQP endpoint to connect to. This could be a send target
      or a receive source.
-    :type remote_address: str, bytes or ~uamqp.address.Address
-    :param auth: Authentication for the connection. This should be one of the subclasses of
-     uamqp.authentication.AMQPAuth. Currently this includes:
-        - uamqp.authentication.SASLAnonymous
-        - uamqp.authentication.SASLPlain
-        - uamqp.authentication.SASTokenAuth
+    :type remote_address: str, bytes or ~pyamqp.endpoint.Target or ~pyamqp.endpoint.Source
+    :param auth: Authentication for the connection. This should be one of the following:
```

Comment:
(self-note for my brain) Okay so we instantiate an AMQP Client only in pyamqp code or in basehandler -- we use Receive and Send in the producer/consumer. Need to check that when we hit the base_handler this gets passed through to the send/recieve client correctly so we can drop hostname in the front-end